src/index.js

```diff
@@ -161,12 +161,20 @@ var audioCtx = new window.AudioContext({
 })
 var arrayBuffer = audioCtx.createBuffer(1, bufferLength, audioCtx.sampleRate)
 
-// keeps track of the active audioSourceNode
-var runningSource = null
+const rampDuration = 0.05
 
-const createNewBeeper = () => {
+// array to hold our dying beepers until they gracefully fade away
+var beepers = []
```

Comment:
at this point we may as well store this in the store

Reply:
You're right, I'll work on a commit for the store.